labs/08_frameworks/A_PyTorch_introduction__autograd_in_action.ipynb

```diff
@@ -60,6 +60,28 @@
     "Refer to https://github.com/peterjc123/pytorch-scripts for Windows installation problems."
```

Comment:
Windows is now supported in the `pytorch` channel so the correct install command is now:
```
conda install -c pytorch pytorch-cpu
```